browser-extension/mv2/src/client/js/ruleExecutionHandler.js

```diff
@@ -2,11 +2,30 @@ RQ.RuleExecutionHandler = {
   appliedRuleIds: new Set(),
 };
 
+RQ.RuleExecutionHandler.sendRuleExecutionEvent = (rule) => {
+  const eventName = "rule_execution";
+  const eventParams = {
+    rule_type: rule.ruleType,
+    rule_id: rule.id,
+    platform: "extension",
+  };
+  eventParams["source"] = "content_script";
```

Comment:
No need for source. `source` does not represent the source of the event here. `platform` is apt for `rule_execution`

Reply:
I don't know, doesn't hurt to add it I guess.